google-cloud-spanner/src/test/java/com/google/cloud/spanner/DefaultBenchmark.java

```diff
@@ -55,17 +54,17 @@
  * <p>Below are a few considerations here: 1. We use all default options for this test because that
```

Comment:
super-nit (feel free to leave as is): Use `<ol>` and `<li>` tags to create an actual ordered (numbered) list.